Payments/UserInterface/01a_ListAuthorizations.md

```diff
@@ -0,0 +1,299 @@
+
+        <!---- Eintrag gibt es nicht mehr [Reset](../../Assets/Icons/Reset.png "[Reset]") reset  
+            Click this entry to reset all unsaved changes to the settings of the selected view. This menu entry is only displayed if a view has been selected and any changes have been made to the view settings.--->
+
+        - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish  
+            Click this entry to publish the view. This menu entry is only displayed if you have selected an unpublished view. As long as you have not published a view, your views are visible for you only.
+
+        - ![Unpublish](../../Assets/Icons/Unpublish.png "[Unpublish]") unpublish  
+            Click this entry to unpublish the view so that it is visible only to you again. This menu entry is only displayed if you have selected and published a view.
+
+        - ![Save](../../Assets/Icons/Save.png "[Save]") save  
+            Click this entry to save the current view settings in the selected view. This menu entry is only displayed if you have selected a view.
+
+            > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+        - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete  
+            Click this entry to delete the selected view. A confirmation window to confirm the deletion is displayed. This menu entry is only displayed if you have selected a view.
+ 
+
+- *Connections*   
+    Click the drop-down list to select a connection. The following options are available:
+    - **All connections**  
+        Select this option to display authorization transactions for all configured connections. 
+    - **Connection name**  
+        Select this option to display authorization transactions for the selected connection. When a specific connection has been selected, the **All connections** option is no longer displayed. Click the selected connection again to deselect it.  
+
+
+
+- ![Search](../../Assets/Icons/Search.png "[Search]") (Search)   
+    Click this button to display the search bar and search for authorization transactions.   
+
+
+- ![Refresh](../../Assets/Icons/Refresh01.png "[Refresh]") (Refresh)   
+    Click this button to update the list of authorization transactions.   
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+    Click this button to display the columns bar and to customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.   
+    > [Info] It is recommended to add the following columns to your view to get the required information and to better filter and sort the entries: Status ID, Status information (if applicable).    
```

Comment:
Diese Info gehört eher in Operation. Hier geht es in erster Linie um die reine Beschreibung der UI Elemente (mit Links zu Integration/Operation, wenn nötig). Info und Caution Boxes sind hier eher selten.

Reply:
Done